### PR TITLE
gityghasemi

### PR DESCRIPTION
**Pull request by gitygity**
this is my code. i hope this pull request be correct

### CODE REVIEW

File: bf_code.b [hunk 1]
```diff
@@ -0,0 +1,177 @@
+[-]
```
Comment (masihtehrani) [inline bf_code.b]: do nothing

**Review by masihtehrani — CHANGES_REQUESTED** [review]
please change file to your_name+your_family.bf
please write comment for any charachter

**Review by masihtehrani — CHANGES_REQUESTED** [review]
for one feature use one commit
for change one feature and commit on last commit use
commit --amend

**Review by ahangarha — APPROVED** [review]
The code is working as expected. But I think there are too many loops.

I also passed a test on your code with my unittest on my own script and it passed :+1:

**Review by masihtehrani — APPROVED** [review]
I approve your code,but you are note resolve 2 commit!

**Pull request merged**